talk/basicconcepts/control.tex

```diff
@@ -271,3 +271,16 @@
     \end{cppcode*}
   \end{alertblock}
 \end{frame}
+
+\begin{frame}[fragile]
+  \frametitlecpp[11]{Control structures}
+  \begin{alertblock}{Exercise}
+    Familiarise yourself with different kinds of control structures. Re-implement them in different ways.
+    \begin{itemize}
+      \item go to \texttt{code/control}
+      \item Look at \texttt{control.cpp}
+      \item Compile it (\texttt{make}) and run the program (\texttt{./control})
+      \item Work on the tasks that you find in \texttt{README.md}
```

Comment:
Inconsistent case after `\item`.
```suggestion
      \item Go to \texttt{code/control}
      \item Look at \texttt{control.cpp}
      \item Compile it (\texttt{make}) and run the program (\texttt{./control})
      \item Work on the tasks that you find in \texttt{README.md}
```

Even though this is generally very inconsistent in the slide deck.